### ISSUE TITLE
Any advantage to Gecko install method

### ISSUE THREAD

**Issue by tlcmd** [geckolinux/geckolinux-project#100]
I've successfully installed GeckoLinux Mate onto my computer with the first install choice. Is there and advantage to be gained by my installing it myself using my partition table over the GeckoLinux install next to the existing OS' on my HDD?

Thanks for all your help and advice and putting up with an rpm illiterate old guy. I appreciate your polite patience with me and so far prefer the GeckoLinux to any prior rolling distro I've tried (and there wee a bunch of them).
tlcmd

**Comment by geckolinux** [comment]
Hi there,

I'd say that if you already have an existing partition that you don't mind blowing away, it would probably make more sense to install GeckoLinux there, rather than carving up your disk even more.

Hope this helps! Cheers.

**Comment by tlcmd** [comment]
GeckoLinux, Thanks for the response. I've already installed GeckoLinux on a single large partition using the "install alongside" option after my attempts to try an install re partitioning  method were an abysmal failure. So I now have Windows 10 (which I hate), Linux Mint 19 Mate which I partitioned and installed, and the above described install of GeckoLinux. All are working using the reFind boot loader. I asked just to see if there were any advantages to my doing (attempting again) a self partitioned install of GeckoLinux.
Thanks again. Now I have to re-install all my files and folders from my external HDD to both Linux distros. That ought to keep me busy for several days so you, with luck, won't have me pestering you again for awhile.

tlcmd
